sui_core/src/checkpoints/proposal.rs

```diff
@@ -0,0 +1,90 @@
+use std::collections::{BTreeMap, HashSet};
```

Comment:
It would be good to include some Readme or documentation block here to explain what the general process is for creating the checkpoints.

Reply:
I will move the notion docs to a README.md in the checkpoint folder.